Incopo/AppDelegate.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AppDelegate.swift
+//  Incopo
+//
+//  Created by Laurentiu Ile on 08.02.2021.
+//
+
+import UIKit
+import CoreData
+import Firebase
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        FirebaseApp.configure()
+        return true
+    }
+
+    // MARK: - Core Data stack
+
+    lazy var persistentContainer: NSPersistentContainer = {
+    
+        let container = NSPersistentContainer(name: "Incopo".localized)
```

Comment:
don't localize this, can cause problems when switching language. it's not displayed on UI so no need for localization